### DISCUSSION TITLE
detecting version...

### DISCUSSION THREAD

**Discussion by madcoda9000** [photoview/photoview#522] [Q&A and Troubleshooting] [answered]
Hello,

when photoview is build manually, is there any possibility to detect which version is installed? So i can decide when i have to update my installation / build.

Greetings
Sascha

**Comment by viktorstrate** [comment, marked as answer]
There is no good way to do that unfortunately, if you're cloning the project with `git` the you can run `git log` to see the latest commit you have pulled.

**Reply by madcoda9000** [reply]
OK, thx
